_posts/2022-10-20-login-issues.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Login issues with newer browsers"
+author: garrett
+tags: cockpit
+---
+
+## Quick summary
+
+Firstly, there's a fix.
+
+In most cases, update. [Cockpit 277](cockpit-277), released last month, has the fix.
+
+Updating Cockpit should work for Arch, CentOS Stream, Debian, Fedora, openSUSE, non-LTS versions of Ubuntu.
+
+If updating doesn't work (in Ubuntu LTS some versions of RHEL, Alma Linux, and Rocky Linux), then hopefully it will work very soon. For the time being, please [look at the bottom of this post for some workarounds](#workarounds).
+
+## Summary
+
+Logging in with new versions of browsers into a few older versions of Cockpit is currently broken due to a check in the login code. This check used to pass in all browsers, but very recently every browser have changed the way they parse the string. Firefox release 106 — released this week — has this parsing change; all other browsers will have it in upcoming versions too.
+
+We fixed this bug as soon as it was discovered (a month ago) and pushed [a new release of Cockpit](cockpit-277) and also backported it as a hotfix to all the "stable" slower-moving distributions.
+
+All distributions have had the fix for a around a month. Most distributions have accepted and published the fix, including Arch Linux, CentOS Stream 8 & 9, Debian, Fedora, openSUSE Tumbleweed, Red Hat Linux Enterprise Linux versions prior to 8.8, and Ubuntu Kenetic.
+
+Unfortunately, some of the slower-moving distributions ("enterprise" and "long term support") haven't updated yet — and this notably includes Red Hat Enterprise Linux 8.6 & 9.0, Ubuntu 22.04 LTS, and Rocky Linux.
```

Comment:
Isn't this a bit over verbose? Maybe just say: "Due to the nature of enterprise/LTS distributions the fixes have not landed yet in X, Y,Z .

Reply:
```suggestion
Due to the nature of enterprise/LTS distributions the fixes have not landed yet in Red Hat Enterprise Linux 8.6 & 9.0, Ubuntu 22.04 LTS, Alma Linux, and Rocky Linux.
```